lib/bulk-apply-applications.js

```diff
@@ -243,91 +261,166 @@ function bulkApplyApplications (args, callback) {
-            } else {
-              eventDate = moment.utc(firstDate.startTime).format(eventDateFormat) + ' ' + startTime + ' - ' + endTime;
-            }
-            var payload = {
+          //set email intro based on application status
+          if (applications[0].status === 'pending') {
+            emailIntro = 'This is a notification to let you know that your request for a ticket for the below event has been received. Once your request has been approved you will receive your ticket confirmation by email.';
+          } else if (applications[0].status === 'approved') {
+            emailIntro = 'This is your order confirmation for the below event.';
+          } else if (applications[0].status === 'cancelled') {
+            emailIntro = 'Your ticket has been cancelled for the below event.';
+          }
+          async.each(profiles, function (profile, profileCb) {
+            //email payload changes for applicants
+            var payloadChanges = {
               to: profile.email || null,
               code: emailCode,
               subject: emailSubject,
-              subjectVariables: [event.name],
-              locality: locality,
               from: dojo.name+' <'+dojo.email+'>',
-              replyTo: dojo.email,
               content: {
                 applicantName: profile.name,
-                event: event,
-                dojo: dojo,
-                applicationDate: moment.utc(applications[0].created).format(eventDateFormat),
-                sessionName: session.name,
-                tickets: tickets,
-                status: applications[0].status,
-                applicationId: applications[iterator] ? applications[iterator].id : null,
-                eventDate: eventDate,
-                cancelLinkBase: protocol + '://' + zenHostname + '/dashboard/cancel_session_invitation'
+                intro: emailIntro
               }
             };
-
+            //create the payload for the applicant
+            var payload = _.merge(payloadChanges, commonPayload);
             iterator++;
-
-            if (!profile.email) return emailParents(cb);
-            seneca.act({role: 'cd-dojos', cmd: 'send_email', payload: payload}, function (err, res) {
-              if (err) return cb(err);
-              if (applications[0].status === 'approved' || applications[0].status === 'pending' || applications[0].status === 'cancelled' ) {
-                emailParents(cb);
+            //If this applicant has no email of their own
+            if (!profile.email) {
+              //move to next applicant if there are no parents for this applicant either (We have no one to email)
+              if (_.isEmpty(profile.parents)) {
+                return profileCb();
               } else {
-                return cb();
+                //we email the parent email if no email is found for this applicant
+                payload.to = profile.parents[0].email || null;
```

Comment:
This create a duplicate email to a parent, as it's now sent by default after i believe it's not required anymore.